.github/workflows/android.yml

```diff
@@ -87,7 +87,7 @@ jobs:
       - name: 🔨 Prepare build env
         run: |
           sudo apt-get update
-          sudo apt-get install -y gperf autopoint '^libxcb.*-dev' libx11-xcb-dev libegl1-mesa libegl1-mesa-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev autoconf-archive libgstreamer-gl1.0-0 libgstreamer-plugins-base1.0-0 libfuse2 nasm
+          sudo apt-get install -y gperf autopoint '^libxcb.*-dev' libx11-xcb-dev libegl1-mesa libegl1-mesa-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev autoconf-archive libgstreamer-gl1.0-0 libgstreamer-plugins-base1.0-0 libfuse2 libcups2-dev nasm
```

Comment:
Could this be built as part of vcpkg or not?

Reply:
Maybe, but it's not there ATM.

An alternative would be to add a proper WITH_PRINTER=ON/OFF in QGIS, which would remove the need for printing altogether which we do not use.

If we want to spend more energy, I'd vote for the latter.

Reply:
Let's stick to what we have here